HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+# CMSSW version specific customizations
+import os
+cmsswVersion = os.environ['CMSSW_VERSION']
+
+if cmsswVersion >= "CMSSW_7_5":
+
```

Comment:
Any problem can be solved in different ways, we picked this way because it
makes our life easier - having as much as possible the same as HLT needs
to work across releases (74 for data taking and HLT menu development, 
75 for CMSSW development related HLT changes, HLT developers
using 74 but newer menus, HLT developers using new menus in 75X,
offline using frozen menus of MC production stuff, etc.)